util/src/builder/command.rs

```diff
@@ -117,19 +135,33 @@ impl CommandBuilder {
     /// Set the localization dictionary for the command description.
     ///
     /// Defaults to [`None`].
-    #[allow(clippy::missing_const_for_fn)]
-    pub fn description_localizations(mut self, localizations: HashMap<String, String>) -> Self {
-        self.0.description_localizations = Some(localizations);
+    pub fn description_localizations<S: Into<String>>(
+        mut self,
+        localizations: impl IntoIterator<Item = (S, S)>,
```

Comment:
We should name this T as it's a standard, or make this a bit clearer and more generic:
```suggestion
    pub fn description_localizations<K: Into<String>, V: Into<String>>(
        mut self,
        localizations: impl IntoIterator<Item = (K, V)>,
```